### PR TITLE
freeswitch 1.10.12

### PR DESCRIPTION
**Pull request by daeho-ro**
This is a fix for 

- #179451
- #178326 

The error only comes from clang 15, where the warning 'int-conversion' is now become an error

- https://www.redhat.com/en/blog/new-warnings-and-errors-clang-15

and so I added simple patch to ignore this one.

Once the reference issue is resolved, the fix should be removed.

- https://github.com/signalwire/libks/issues/204
----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/f/freeswitch.rb [hunk 1]
```diff
@@ -114,8 +114,11 @@ class Freeswitch < Formula
 
   resource "libks" do
     url "https://github.com/signalwire/libks.git",
-        tag:      "v2.0.2",
-        revision: "423c68c92d1871e2c9cb83d974eac7830ddcdcf5"
+        tag:      "v2.0.6",
+        revision: "3bc8dd0524a865becdd98c3806735eb306fe0a73"
+
+    # Fix compile with newer Clang, https://www.redhat.com/en/blog/new-warnings-and-errors-clang-15
+    patch :DATA if DevelopmentTools.clang_build_version >= 1500
```
Comment (carlocab) [inline Formula/f/freeswitch.rb:121]: Thanks, can you send this patch upstream? (Or have you done that already?)

Comment (daeho-ro) [reply Formula/f/freeswitch.rb:121]: - https://github.com/signalwire/libks/issues/217

Noted. :)

**Review by carlocab — APPROVED** [review]
Thanks! Can you update the comment on the patch with a link to the issue with your patch in a follow-up?

**Comment by daeho-ro** [comment]
@carlocab You mean not redhat but the github issue, right? I will.

**Comment by carlocab** [comment]
Yes, thanks!

**Pull request merged**